src/client/pages/CountryHome/Repository/EditForm/InputFieldFile/SelectedFile/SelectedFile.tsx

```diff
@@ -3,22 +3,23 @@ import React from 'react'
 
 import { RepositoryItem } from 'meta/cycleData'
 
-import { useRepositoryItem } from 'client/store/ui/repository'
+import { useRepositoryFile, useRepositoryItem } from 'client/store/ui/repository'
 import ButtonDelete from 'client/components/Buttons/ButtonDelete'
 import { useOnRemoveFile } from 'client/pages/CountryHome/Repository/EditForm/InputFieldFile/hooks/useOnRemoveFile'
 import RepositoryLink from 'client/pages/CountryHome/Repository/RepositoryLink'
 
 const SelectedFile: React.FC = () => {
   const onRemoveFile = useOnRemoveFile()
   const repositoryItem = useRepositoryItem()
+  const repositoryFile = useRepositoryFile()
 
   if (!repositoryItem) {
     return null
   }
 
   return (
     <div className="repository-form__selected-file">
-      <RepositoryLink datum={repositoryItem as RepositoryItem} />
+      <RepositoryLink datum={repositoryItem as RepositoryItem} name={repositoryFile.name} />
```

Comment:
In select file we want to show the file name that the user has selected, and not the repository item name that is _public_